linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/routers.js

```diff
@@ -36,23 +36,22 @@ var Routers = (function() {
   var clientRE = /^rt\/(.+)\/dst\/id\/(.*)\/requests$/,
       serverRE = /^rt\/(.+)\/srv\/([^/]+)\/(\d+)\/requests$/;
 
+  var colorIdx = 0;
```

Comment:
I'd move this inside mkColor
no reason for the rest of the routers code to access it